packages/core/test/driver.test.ts

```diff
@@ -625,6 +625,8 @@ describe('Driver', () => {
       mode: 'WRITE',
       reactive: false,
       impersonatedUser: undefined,
+      // @ts-expect-error
+      log: driver?._log,
```

Comment:
Why is this? I don't understand it.

Reply:
`_driver` is not exposed as public api, so I have to tell typescript this is an type error, but this is okay.